engine/src/main/java/org/terasology/rendering/dag/nodes/WorldReflectionNode.java

```diff
@@ -100,8 +101,8 @@ public void process() {
         playerCamera.setReflected(false);
 
         GL11.glCullFace(GL11.GL_BACK);
-        bindDisplay();
-        setViewportToSizeOf(sceneOpaque);
+        bindDisplay(); // TODO: remove since its reset by default BindFBO state change
```

Comment:
It doesn't matter much for this particular PR, but how come you removed the bindDisplay line in the ShadowMapNode and you left it here?